api.swagger.yaml

```diff
@@ -6361,9 +6361,15 @@ paths:
             items:
               type: string
               enum:
-                - image
-                - document
                 - audio
```

Comment:
The example schema is showing '.jpg' under audio, I'm guessing it's because it's first in this list after image was removed. Can we change the example to an audio file that we allow?

Reply:
changed examples to audio